src/model/mitra-activity.ts

```diff
@@ -1,10 +1,10 @@
 import { ActivityObject, Link } from "activitypub-objects";
-import { Activity as ActivityExt } from "activitypub-objects/dst/activities/activity";
+import { Activity } from "activitypub-objects/dst/activities/activity";
 
 type Modify<T, R> = Omit<T, keyof R> & R;
 
-export type Activity = Modify<
-  ActivityExt,
+export type ActivityImplementation = Modify<
```

Comment:
sorry for leading you to the wrong path. This `Implementation` was only to demostrate how I would structure it. Name convention wise I would prefer to suffix the interfaces with `Interface` and name the implementation just like the interface without the suffix: 
```
export type Activity = Modify<ActivityInterface, ...
```